### PR TITLE
Add meeting notes from Sep 9-14 2020

### PR DESCRIPTION
**Pull request by michdolan**
Includes notes from last TSC meeting, ACES config, and Documentation working groups.

Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

**Review by doug-walker — APPROVED** [review]
Thank you so much for taking these notes Michael, super useful!

**Comment by zachlewis** [comment]
Strongly agreed; as a regular meeting-misser myself, I really appreciate the effort. It's a lot of work. I felt like a fly on the wall in that alias discussion in particular -- I like that it kind of ends on a cliffhanger.  And as a first-time meeting-attender, it's really helpful and interesting seeing my participation reflected in the minutes.

**Comment by hodoulp** [comment]
Approve the TSC meeting minutes.

File: ASWF/meetings/working_groups/configs/2020-09-08.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+July 28, 2020
```
Comment (Simran-B) [inline ASWF/meetings/working_groups/configs/2020-09-08.md:4]:
```suggestion
September 8, 2020
```

File: ASWF/meetings/working_groups/documentation/2020-09-11.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+June 19, 2020
```
Comment (Simran-B) [inline ASWF/meetings/working_groups/documentation/2020-09-11.md:4]:
```suggestion
September 11, 2020
```

File: ASWF/meetings/working_groups/documentation/2020-09-11.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+Rotating Secretary: Michael Dolan
+
+Attendees:
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Zach Lewis - Method
+  * [X] Christophe Brejon
+  * [X] Simran Brucherseifer
```
Comment (Simran-B) [inline ASWF/meetings/working_groups/documentation/2020-09-11.md:16]: Got married 😁 
```suggestion
  * [X] Simran Spiller
```

**Review by Simran-B — CHANGES_REQUESTED** [review]
Always appreciated indeed, thanks Michael!

File: ASWF/meetings/working_groups/documentation/2020-09-11.md (hunk 3 above)
Comment (michdolan) [reply ASWF/meetings/working_groups/documentation/2020-09-11.md:16]: Congrats! I'll get these revisions made today.

File: ASWF/meetings/tsc/2020-09-14.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [X] Mark Titchener (_TSC_) - Foundry
+  * [ ] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * Bernard Lefebvre - Autodesk
+  * Alessandra Tomassi - Industrial Light & Magic
+  * Matthias Scharfenberg - Industrial Light & Magic
+  * J Schulte - Industrial Light & Magic
```
Comment (Simran-B) [inline ASWF/meetings/tsc/2020-09-14.md:23]: Are they `[x]` or `[ ]`?

Comment (michdolan) [reply ASWF/meetings/tsc/2020-09-14.md]: Fixed here and for the previous week.

**Review by hodoulp — APPROVED** [review]
The pull request is clearly a 'small changes' one and there is already one approval so, I approve it.

**Comment by hodoulp** [comment]
@michdolan The PR is ready to merge.

**Pull request merged**